packages/components/cascader-panel/src/index.vue

```diff
@@ -128,8 +128,8 @@ export default defineComponent({
         }
       }
 
-      oldConfig = configTemp
-      oldOptions = options
+      oldConfig = JSON.parse(JSON.stringify(configTemp))
```

Comment:
Maybe we can use `_.cloneDeep` from lodash.

Reply:
ok

Reply:
it changed